ci/build_wheel.sh

```diff
@@ -23,7 +23,7 @@ pyproject_file="${package_dir}/pyproject.toml"
 
 sed -i "s/^name = \"${package_name}\"/name = \"${package_name}${PACKAGE_CUDA_SUFFIX}\"/g" ${pyproject_file}
 echo "${version}" > VERSION
-sed -i "/^__git_commit__/ s/= .*/= \"${commit}\"/g" "${package_dir}/${package_name}/_version.py"
+sed -i "/^__git_commit__/ s/= .*/= \"${commit}\"/g" "${package_dir}/${package_name//-/_}/_version.py"
```

Comment:
I think the better fix here would be to rename the directory `python/dask_cudf` to `python/dask-cudf`, to align with the package name. However, that is more breaking. It would also affect `cudf_kafka`. It's probably good to separate that discussion from this PR since its scope is considerably larger.

Reply:
I don't have strong feelings about this. I just don't think there's a good reason to favor a string replacement in a script (more of a workaround) over the principle of matching the directory structure to the package names. The change would come with some nonzero cost and potential for breakage, though, so I wouldn't do it lightly.

Reply:
FWIW, the change here changes `python/dask_cudf/dask-cudf` => `python/dask_cudf/dask_cudf` basically fixing the module name.

But I think you're right that it should be `python/$package_name/$module_name` and therefore `python/dask-cudf/dask_cudf`.